docs/man/borg-recreate.1

```diff
@@ -160,7 +160,7 @@ consider archives older than (now \- TIMESPAN), e.g. 7d oder 12m.
 consider archives newer than (now \- TIMESPAN), e.g. 7d or 12m.
 .TP
 .BI \-\-target \ TARGET
-create a new archive with the name ARCHIVE, do not replace existing archive (only applies for a single archive)
+create a new archive with the name TARGET, do not replace existing archive (only applies for a single archive)
```

Comment:
Ah, good catch! Thanks for finding this.

But, this is not the right place to fix it because these files are automatically generated from the files in `src/borg/archiver/*.py`, so you need to find/fix the string there (master branch).

In 1.2-maint branch, it is in `src/borg/archiver.py`

Reply:
OK! Should I close this PR?

Reply:
yes, guess it is easiest to make a new PR.